### ISSUE TITLE
Finetunning encoder model for individual tibetan text layer(Yigchung)  detection.

### ISSUE THREAD

**Issue by karmatai** [OpenPecha/BEC#423]
## Description
In this card, we are going to finetune encoder model for one tibetan text layer( yigchung) and compare the result with the previous multi tibetan text layer model to see if it does better than the previous one.

## Subtask
- [ ] Prepare training pipeline and training script to finetune encoder model for yigchung detection.
- [ ] Set up vast ai and run training script.
- [ ] Evaluate, compare and document result.

## Reviewer
- [ ] @kaldan007

**Comment by karmatai** [comment]
Change the layer from yighucng to quoatation. Include all the new datas which are converted by monlam in the training data.

**Comment by karmatai** [comment]
<img width="1280" height="864" alt="Image" src="https://github.com/user-attachments/assets/078691ea-db3a-4761-938a-39f90462ff3d" />
Text: ལས། ནས།
Person: ཞལ་ནས། རྡོ་རྗེས།
MISC: ཇི་སྐད་དུ།

**Comment by karmatai** [comment]
**Dataset**: https://huggingface.co/datasets/karma689/tibetan-quotation-detection
**Model**: https://huggingface.co/karma689/tibetan-quotation-detection
**Result**:
| Metric | Test |
|---|---:|
| IoU@0.5 F1 | **0.853** |
| Precision | 0.831 |
| Recall | 0.876 |
| Pred / gold spans | 2423 / 2299 |
